src/main/java/com/keniu/repositories/OrderRepository.java

```diff
@@ -0,0 +1,14 @@
+package com.keniu.repositories;
+
+import com.keniu.models.Order;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.jpa.repository.EntityGraph;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface OrderRepository extends JpaRepository<Order, Long> {
+
+    @EntityGraph(attributePaths = "user")
+    Page<Order> findByUserId(Long userId, Pageable pageable);
+
```

Comment:
```suggestion
    @EntityGraph(attributePaths = "user")
    Page<Order> findByUserId(Long userId, Pageable pageable);
```

Reply:
@liliia-ponomarenko 
I don't understand what you're suggesting. Please explain

Reply:
Deleted empty rows